src/Http/Cookie/Cookie.php

```diff
@@ -245,20 +236,27 @@ protected function validateName($name)
     }
 
     /**
-     * Gets the cookie value
-     *
-     * @return string|array
+     * {@inheritDoc}
      */
     public function getValue()
     {
         return $this->value;
```

Comment:
I applied your fix and it correctly save the cookie now.
However when I read the cookie value, it continues to be in json format.

Can be correct to modify here like this?
```php
return $this->_expand($this->value);
```

But also when using `ServerRequest->getCookie($cookie_name);` it returns the cookie value in json format and it doesn't call `Cookie->getValue()`.

Reply:
By setting
```php
return $this->_expand($this->value);
```
There are warnings in the `EncryptedCookieMiddleware` so it's more complicated than this :sweat_smile:

Reply:
The value of the cookie is the JSON data though when you're reading from a request. The cookie middleware adds more behavior to cookies though. I wouldn't recommend people store JSON data in plaintext cookies as its full of 💣